cmd/sonobuoy/app/master.go

```diff
@@ -27,47 +28,57 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var noExit bool
-var kubecfg Kubeconfig
+type aggregatorInput struct {
+	noExit  bool
+	kubecfg Kubeconfig
+}
 
-func NewCmdMaster() *cobra.Command {
+// NewCmdAggregator returns the command that runs Sonobuoy as an aggregator. It will
+// load the config, launch plugins, gather results, and query the cluster for data.
+func NewCmdAggregator() *cobra.Command {
+	input := aggregatorInput{}
 	cmd := &cobra.Command{
-		Use:    "master",
-		Short:  "Runs the master/aggregator component (for internal use)",
-		Long:   "Sonobuoy is an introspective kubernetes component that generates reports on cluster conformance, configuration, and more",
-		Run:    runMaster,
+		Use:   "aggregator",
+		Short: "Runs the aggregator component (for internal use)",
+		Long:  "Sonobuoy is an introspective kubernetes component that generates reports on cluster conformance, configuration, and more",
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Didn't want to change this to `RunE` but will do that with a bunch of commands at the same time eventually. We should be writing our code in a way that allows us to bubble up errors to the main `Execute` method so we can centralize error handling rather than having logging/`os.Exit` all over the place.